index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Skeleton project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+
+  <body>
+    <header>
+      <section class="navigator1">
+        <div class="navigator">
+          <span class="mylogo">My logo</span>
+          <span><img src="images/Enabled.png" alt="" class="hamburger"></span>
+        </div>
+
+      </section>
+    </header>
+
+    <main class="container">
+      <section class="container1">
+        <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">LET’S CONNECT</a>
+        <div class="social-media-container" role="region">
+          <i class="i-angel"><a href="#"><img src="images/angellist.png" alt=""></a></i>
+          <i class="i-twitter"><a href="#"><img src="images/twitter.png" alt=""></a></i>
+          <i class="i-github"><a href="#"><img src="images/github.png" alt=""></a></i>
+          <i class="i-medium"><a href="#"><img src="images/medium.png" alt=""></a></i>
+        </div>
```

Comment:
- [ ] Consider the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).